src/components/vm/nics/nicAdd.tsx

```diff
@@ -184,6 +196,9 @@ export class AddNIC extends React.Component<AddNICProps, AddNICState> {
             else
                 this.setState({ networkSource: "", saveDisabled: true });
         }
+
+        if (this.state.doOnlineValidation)
+            this.setState(state => ({ validation: validateDialogBodyValues(state) }));
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2331-3e464b33-20251031-114223-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)